test/test_quantize_script.py

```diff
@@ -71,5 +72,101 @@ def forward(self, x):
                    .check_not('Observer = prim::GetAttr[name="_observer_') \
                    .run(m.graph)
 
+
+    def test_insert_quant_dequant_dynamic(self):
+        class M(torch.nn.Module):
+            def __init__(self):
+                super(M, self).__init__()
+                self.conv = torch.nn.Conv2d(3, 5, 3).float()
+
+            def forward(self, x):
+                return self.conv(x)
+
+        m = torch.jit.script(M())
+        qconfig = default_dynamic_qconfig
+        qconfig_dict = {
+            '': script_qconfig(qconfig)
+        }
+        m = wrap_cpp_module(torch._C._jit_pass_insert_observers(m._c, "forward", qconfig_dict, False, True))
+        data = torch.randn(1, 3, 10, 10, dtype=torch.float)
+
+        get_forward(m._c)(data)
+
+        m = wrap_cpp_module(torch._C._jit_pass_insert_quant_dequant(m._c, "forward", False, True))
```

Comment:
same for this one, `convert_script`

Reply:
I've added `convert_dynamic_script` in the next PR so I will update it there.